src/Benchmarks/runtimeconfig.template.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I agree with @dougbu, but I think we should fix this in a separate PR.  Issue: https://github.com/aspnet/benchmarks/issues/184